xml/System/String.xml

```diff
@@ -2069,7 +2072,7 @@ You can download the [Sorting Weight Tables](https://www.microsoft.com/download/
  The following example compares two strings in three different ways: using linguistic comparison for the en-US culture; using linguistic case-sensitive comparison for the en-US culture; and using an ordinal comparison. It illustrates how the three methods of comparison produce three different results.  
   
  [!code-cpp[System.String.Compare#1](~/samples/snippets/cpp/VS_Snippets_CLR_System/system.String.Compare/cpp/example.cpp#1)]
- [!code-csharp[System.String.Compare#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.String.Compare/cs/Example.cs#1)]
+ [!code-csharp-interactive[System.String.Compare#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.String.Compare/cs/Example.cs#1)]
```

Comment:
we should undo this change to the sample as well. It relies on CultureInfo. 

One tip is to test out the inside of the Main method ahead of time on any page that has try .NET enabled and see if it would run and give you the same expected result.

Reply:
Done.